src/Umbraco.Web.UI.Client/src/views/users/views/users/users.html

```diff
@@ -143,10 +143,17 @@
                     <umb-dropdown class="pull-right" ng-if="vm.page.showGroupFilter" on-close="vm.page.showGroupFilter = false;">
                         <umb-dropdown-item ng-repeat="userGroup in vm.userGroups" style="padding: 8px 20px 8px 16px;">
                             <div class="flex items-center">
-                                <umb-checkbox model="userGroup.selected"
-                                              on-change="vm.setUserGroupFilter(userGroup)"
-                                              text="{{ userGroup.name }}">
+                                <umb-checkbox input-id="usergroup-{{$index}}"
+                                              name="usergroup"
+                                              model="userGroup.selected"
+                                              on-change="vm.setUserGroupFilter(userGroup)">
                                 </umb-checkbox>
+                                <label for="usergroup-{{$index}}">
+                                    <span class="flex items-center">
+                                        <i class="{{userGroup.icon}}" aria-hidden="true"></i>
```

Comment:
Can we use the `umb-icon` directive here?

Reply:
@poornimanayar yes, but it didn't seem to work well here at the moment.

> However I haven't used `<umb-icon>` here at the moment since with the latest changes this seems to be a be slow and these icons are first loaded quite a while after the user group filter has been opened.

Reply:
Sorry @bjarnef  missed my eye! I have tested this and looks okay!